packages/react-components/src/disclosure/src/Disclosure.tsx

```diff
@@ -1,12 +1,14 @@
 import "./Disclosure.css";
 
-import { Children, ComponentProps, ElementType, ForwardedRef, KeyboardEvent, MouseEvent, ReactElement, ReactNode, SyntheticEvent, useCallback } from "react";
+import { Children, ComponentProps,ElementType, ForwardedRef, KeyboardEvent, MouseEvent, ReactElement, ReactNode, SyntheticEvent, forwardRef, useCallback } from "react";
 import { DisclosureContext } from "./DisclosureContext";
-import { DomProps, Keys, augmentElement, cssModule, forwardRef, isNil, mergeProps, resolveChildren, useControllableState, useEventCallback, useId, useMergedRefs } from "../../shared";
+import { DomProps, Keys, augmentElement, cssModule, isNil, mergeProps, resolveChildren, useControllableState, useEventCallback, useId, useMergedRefs } from "../../shared";
 import { Text } from "../../typography";
 import { useSlidingTransition } from "./useSlidingTransition";
 
-export interface InnerDisclosureProps extends DomProps {
+const defaultElement = "div";
+
+export interface InnerDisclosureProps extends DomProps, Omit<ComponentProps<typeof defaultElement>, "color"> {
```

Comment:
Do we still need the "DomProps" interface now that the inner props extends ComponentProps? DomProps only contains an optionnal "id" prop